ods_ci/tests/Resources/OCP.resource

```diff
@@ -332,3 +332,18 @@ Wait For Namespace To Be Active
     Log    ${rc}
     Log    ${value}
     Should Be Equal As Integers    ${rc}    0
+
+Get Cluster Type By Domain
```

Comment:
As this keyword is returning a boolean it should be called **Is Self Managed Cluster Domain** or similar to follow the current naming in this file

Reply:
Renamed to `Is Cluster Type Self-Managed` in resemblance to related keyword `Skip If Cluster Type Is Self-Managed`

Reply:
`I don't think you do it in this PR but please don't remove the existing keyword Skip If RHODS Is Self-Managed. It is being used in some tests`
@jgarciao I didn't remove the `Skip If RHODS Is Self-Managed` keyword in this PR. Also, not planning to remove it.
I got it statement regarding this keyword led this interpretation. 
Modifying the statement in description now

Reply:
No problem. Thanks a lot!